test/system/src/main/java/io/pravega/test/system/framework/Utils.java

```diff
@@ -89,23 +89,26 @@ public static Service createBookkeeperService(final URI zkUri) {
         }
     }
 
-    public static Service createPravegaControllerService(final URI zkUri, String serviceName) {
+    public static Service createPravegaControllerService(final URI zkUri, String serviceName, boolean secure) {
         switch (EXECUTOR_TYPE) {
```

Comment:
Does it make sense to log the inputs here, for making it easier to debug system test issues? That is, if its not already logged in the call hierarchy.